### PR TITLE
Add lesson.md file for open data discussion

### PR DESCRIPTION
**Pull request by derekhoward**

### CODE REVIEW

File: lessons/misc/open-data-intro/lesson.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+## How to open your data
+- Metadata
+- Create a DATA_README
+- Use the [data reuse checklist](https://mozillascience.github.io/checklist/)
+
+
+## Resources
+https://mozillascience.github.io/open-data-primers/
+https://mozillascience.github.io/open-data-guides/
+http://mozillascience.github.io/working-open-workshop/data_reuse/
```
Comment (QuLogic) [inline lessons/misc/open-data-intro/lesson.md]: https

File: lessons/misc/open-data-intro/lesson.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+---
+
+ - **Authors**: Derek Howard
+ - **Lesson topic**: A guided discussion on opening up your research data
+
+
+## What is open data?
+[“Open data is data that can be freely used, re-used and redistributed by anyone - subject only, at most, to the requirement to attribute and sharealike.”](http://opendatahandbook.org/guide/en/what-is-open-data/)
+
+Specifically:
```
Comment (QuLogic) [inline lessons/misc/open-data-intro/lesson.md]: Since this whole section is from the above link, maybe this citation should be a bit more explicit.

**Comment by QuLogic** [comment]
Is everything here now? We should probably copy in the few notes that @mbonsma made as well.

**Comment by derekhoward** [comment]
I think it's good to go now. Just let me know if there should be any more additions.

File: lessons/misc/open-data-intro/lesson.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+---
+
+ - **Authors**: Derek Howard
+ - **Lesson topic**: A guided discussion on opening up your research data
+
+
+## What is open data?
+[“Open data is data that can be freely used, re-used and redistributed by anyone - subject only, at most, to the requirement to attribute and sharealike.”](http://opendatahandbook.org/guide/en/what-is-open-data/)
+
+More specifically, from the open data handbook and the [opendefinition.org](http://opendefinition.org/od/2.0/en/):
```
Comment (QuLogic) [inline lessons/misc/open-data-intro/lesson.md]: Latest definition is 2.1; not sure the differences though.

File: lessons/misc/open-data-intro/lesson.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+[“Open data is data that can be freely used, re-used and redistributed by anyone - subject only, at most, to the requirement to attribute and sharealike.”](http://opendatahandbook.org/guide/en/what-is-open-data/)
+
+More specifically, from the open data handbook and [opendefinition.org](http://opendefinition.org/od/2.1/en/):
+- Availability and Access: the data must be available as a whole and at no more than a reasonable reproduction cost, preferably by downloading over the internet. The data must also be available in a convenient and modifiable form.
+- Re-use and Redistribution: the data must be provided under terms that permit re-use and redistribution including the intermixing with other datasets.
+- Universal Participation: everyone must be able to use, re-use and redistribute - there should be no discrimination against fields of endeavour or against persons or groups. For example, ‘non-commercial’ restrictions that would prevent ‘commercial’ use, or restrictions of use for certain purposes (e.g. only in education), are not allowed.
+
+## [Why open data?](https://mozillascience.github.io/open-data-primers/1.1-what-and-why.html)
+- Maintain accountability
```
Comment (lwjohnst86) [inline lessons/misc/open-data-intro/lesson.md]: A minor note. These items in the list all convert into a single paragraph. So it looks like `Maintain accountability When data is open`. I'd suggest adding a colon to the end.

Comment (derekhoward) [reply lessons/misc/open-data-intro/lesson.md]: Good catch! I realised I should actually use a markdown previewer to help me check things before pushing changes :)

**Pull request merged**